api/src/main/java/org/bahmni/module/bahmni/ie/apps/service/BahmniFormService.java

```diff
@@ -20,4 +21,8 @@
     ExportResponse formDetailsFor(List<String> formUuids);
 
     BahmniFormResource saveFormNameTranslation(BahmniFormResource bahmniFormResource, String referenceFormUuid);
+
+    Form getFormDetailsFromFormName(String formName, String formVersion);
```

Comment:
@buvaneswari-arun can _getFormDetailsFromFormName_ changed to _getFormDetails**For**FormName_ ?

Reply:
Will take this up as part of next build as it has needs extensive testing. Followup card created.